doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -52,66 +63,48 @@ The default notification schema installed for Kaa applications is empty. For the
 }
 ```
 
-You can configure your own notification schema via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/saveNotificationSchema) call or use Administration UI as shown below. 
-
-
-The list of notification schemas created by a tenant developer for the application is shown in the **Notification schemas** window.
```

Comment:
Because it's too obvious even for unexperienced users. When we create a new item, of course we expect it to appear in the list of the same type items. It doesn't deserve a paragraph and a screenshot. Plus, it's redundant because the 'Notification schema' section contains the screenshot of that list.